tests/plugins/notifications/tst_ClickAppNotifications.qml

```diff
@@ -1,117 +0,0 @@
-/*
```

Comment:
Is this not needed anymore?

Reply:
It isn't doing anything as the entire block of test functions were commented out. So it makes sense to remove it.